pkg/registry/apis/ofrep/register.go

```diff
@@ -64,10 +64,6 @@ func NewAPIBuilder(providerType string, url *url.URL, insecure bool, caFile stri
 }
 
 func RegisterAPIService(apiregistration builder.APIRegistrar, cfg *setting.Cfg) (*APIBuilder, error) {
-	if !cfg.OpenFeature.APIEnabled {
```

Comment:
Third time is a charm. I have tried removing this `if` conditional twice now. This time, however, it covers our use-case without breaking existing ones.